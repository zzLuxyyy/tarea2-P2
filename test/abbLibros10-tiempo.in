alturaABBLibrosTiempo 100000 1
Fin
