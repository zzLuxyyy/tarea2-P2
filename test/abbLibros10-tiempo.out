1>La altura del arbol es 17. Calculado correctamente en menos de 1s.
2>Fin.
